pages/common/git-contrib.md

```diff
@@ -0,0 +1,9 @@
+# git contrib
+
+> Display commits from a specific author.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-contrib>.
+
+- Display all commit hashes and their corresponding commit messages from a specific author:
+
+`git contrib {{username}}`
```

Comment:
Based on the `git-log.md` page, we used `author` as a placeholder for `first name`, `full name`, `email` or `username`:
```suggestion
`git contrib {{author}}`
```